lib/internal/test_runner/test.js

```diff
@@ -74,8 +75,9 @@ const kHookNames = ObjectSeal(['before', 'after', 'beforeEach', 'afterEach']);
 const kUnwrapErrors = new SafeSet()
   .add(kTestCodeFailure).add(kHookFailure)
   .add('uncaughtException').add('unhandledRejection');
-const { testNamePatterns, testOnlyFlag } = parseCommandLine();
+const { testNamePatterns, testOnlyFlag, bail } = parseCommandLine();
 let kResistStopPropagation;
+let bailedOut = false;
```

Comment:
Should this not be scoped to the root test etc?
What would happen if I called `run({ ...., bail: true })` twice and one has a test which fails?